cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1787,6 +1788,15 @@ public String getVariable(String name) {
         return variables == null ? Collections.<String>emptySet() : variables.keySet();
     }
 
+    /** {@link CallbackKind}s supported by {@link AuthenticationCallbackHandler} */
+    private static final Set<CallbackKind> SUPPORTED_CALLBACK_KINDS = new HashSet<>();
```

Comment:
@honza889 , thank-you for the mapping, it seems that CHOICE is not needed, from javadoc: (not including {@link RealmChoiceCallback}). REALM kind should cover both.